traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -429,6 +787,198 @@ func Create(w http.ResponseWriter, r *http.Request) {
 
 // Used by PUT requests to `/jobs`, replaces an existing content invalidation job
 // with the provided request body.
+func UpdateV40(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	var oFQDN string
+	var dsid uint
+	var uid uint
+	job := tc.InvalidationJobV40{}
+	row := inf.Tx.Tx.QueryRow(putInfoQueryV40, inf.Params["id"])
```

Comment:
I'm thinking `id` needs to be a `requiredParam` and `intParam` in the call to `api.NewInfo`.